src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -945,6 +941,23 @@ private StringBuilder getFeedbackSessionResultsForQuestionInCsvFormat(
         return exportBuilder;
     }
 
+    private int getMaxNumberOfResponseComments(List<FeedbackResponseAttributes> allResponses,
+                                               Map<String, List<FeedbackResponseCommentAttributes>> responseComments) {
```

Comment:
Hi, @chowyb Thanks for the review. Checkstyle forced me to use 12 spaces instead of 8.